Dockerfile

```diff
@@ -77,5 +80,6 @@ EXPOSE 5000
 
 ENV COMPUTE_BACKENDS $COMPUTE_BACKENDS
 ENV FLASK_APP reana_job_controller/app.py
+ENV REANA_LOG_LEVEL DEBUG
```

Comment:
It should not be necessary to hard-code `REANA_LOG_LEVEL` in `Dockerfile`.  One could pass it as environment variable...

Reply:
Just to add a bit more information on this: the way of passing this configuration is a bit cumbersome right now as one would have to: go inside RWC, read the config value and pass it as an environment variable to RJC. In https://github.com/reanahub/reana/issues/277 there is a description of the current process and possible solutions we could take to centralise these configuration variables.

Reply:
Yes this should have been taken out, I missed this when cleaning things up.